projects/1.16/assets/mystical-world/mysticalworld/lang/zh_cn.json

```
@@ -1 +1,434 @@
-{}
\ No newline at end of file
+{
+    "advancement.mysticalworld.aubergine": "那里没有胡萝卜……",
+    "advancement.mysticalworld.aubergine.desc": "紫色胡萝卜曾经风靡一时，但它是茄子",
+    "advancement.mysticalworld.epic_squid": "史诗般的鱿鱼！",
+    "advancement.mysticalworld.epic_squid.desc": "品尝一下紫色史诗鱿鱼的美味吧",
+    "advancement.mysticalworld.root": "魔幻世界",
+    "advancement.mysticalworld.root.desc": "魔法无处不在！",
+    "advancement.mysticalworld.sapphire": "偶然寻到的蓝宝石",
+    "advancement.mysticalworld.sapphire.desc": "你有着足够的幸运来找到一颗比钻石更好的宝石",
+    "attribute.name.forge.swimSpeed": "游泳速度",
+    "attribute.name.generic.reachDistance": "触及半径",
+    "attribute.name.mysticalworld.blessed": "祝福",
+    "attribute.name.mysticalworld.serendipity": "意外惊喜",
+    "attribute.name.mysticalworld.smite": "亡灵杀手",
+    "block.mysticalworld.aubergine_crop": "茄子",
+    "block.mysticalworld.blackened_stone": "漆黑石",
+    "block.mysticalworld.blackened_stone_slab": "漆黑石台阶",
+    "block.mysticalworld.blackened_stone_small_post": "漆黑石窄柱",
+    "block.mysticalworld.blackened_stone_stairs": "漆黑石楼梯",
+    "block.mysticalworld.blackened_stone_wall": "漆黑石墙",
+    "block.mysticalworld.blackened_stone_wide_post": "漆黑石宽柱",
+    "block.mysticalworld.bone_pile_1": "骨堆",
+    "block.mysticalworld.bone_pile_2": "骨堆",
+    "block.mysticalworld.bone_pile_3": "骨堆",
+    "block.mysticalworld.bone_pile_4": "骨堆",
+    "block.mysticalworld.brown_mushroom": "棕色蘑菇",
+    "block.mysticalworld.brown_mushroom_fence": "棕色蘑菇栅栏",
+    "block.mysticalworld.brown_mushroom_fence_gate": "棕色蘑菇栅栏门",
+    "block.mysticalworld.brown_mushroom_full": "完整的棕色蘑菇",
+    "block.mysticalworld.brown_mushroom_slab": "棕色蘑菇台阶",
+    "block.mysticalworld.brown_mushroom_small_post": "棕色蘑菇窄柱",
+    "block.mysticalworld.brown_mushroom_stairs": "棕色蘑菇楼梯",
+    "block.mysticalworld.brown_mushroom_wall": "棕色蘑菇墙",
+    "block.mysticalworld.brown_mushroom_wide_post": "棕色蘑菇宽柱",
+    "block.mysticalworld.charred_fence": "烧焦木栅栏",
+    "block.mysticalworld.charred_fence_gate": "烧焦木栅栏门",
+    "block.mysticalworld.charred_log": "烧焦原木",
+    "block.mysticalworld.charred_planks": "烧焦木板",
+    "block.mysticalworld.charred_slab": "烧焦木台阶",
+    "block.mysticalworld.charred_small_post": "烧焦木窄柱",
+    "block.mysticalworld.charred_stairs": "烧焦木楼梯",
+    "block.mysticalworld.charred_wall": "烧焦木墙",
+    "block.mysticalworld.charred_wide_post": "烧焦木宽柱",
+    "block.mysticalworld.charred_wood": "烧焦木",
+    "block.mysticalworld.copper_block": "铜块",
+    "block.mysticalworld.copper_ore": "铜矿石",
+    "block.mysticalworld.copper_slab": "铜台阶",
+    "block.mysticalworld.copper_small_post": "铜窄柱",
+    "block.mysticalworld.copper_stairs": "铜楼梯",
+    "block.mysticalworld.copper_wall": "铜墙",
+    "block.mysticalworld.copper_wide_post": "铜宽柱",
+    "block.mysticalworld.gall_apple_crop": "瘿苹果",
+    "block.mysticalworld.granite_quartz_ore": "花岗岩石英矿石",
+    "block.mysticalworld.iron_brick": "铁砖块",

```

```suggestion
    "block.mysticalworld.iron_brick": "铁砖",
```
如果没有对应铁砖物品的话最好不要`块`这个字（原版应该也是这样的）